openspec/changes/add-dev4-admin-reports/tasks.md

```
@@ -169,16 +200,38 @@
 
 ### 7.1 Admin DTOs
-- [ ] 7.1.1 Create `admin-users.dto.ts` in `shared/types/src/dtos/`
-- [ ] 7.1.2 Create `admin-entities.dto.ts` in `shared/types/src/dtos/`
-- [ ] 7.1.3 Create `admin-assignments.dto.ts` in `shared/types/src/dtos/`
-- [ ] 7.1.4 Create `admin-reports.dto.ts` in `shared/types/src/dtos/`
-- [ ] 7.1.5 Create `month-locks.dto.ts` in `shared/types/src/dtos/`
-- [ ] 7.1.6 Export all DTOs from `shared/types/src/index.ts`
+- [x] 7.1.1 Create `admin-users.dto.ts` in `shared/types/src/dtos/`
+- [x] 7.1.2 Create `admin-entities.dto.ts` in `shared/types/src/dtos/`
+- [x] 7.1.7 Update `ProjectDto` to include `assignedUsers` array:
+  ```typescript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)